.gitattributes

```diff
@@ -1,4 +1,4 @@
-/tests/
+/tests/ export-ignore
```

Comment:
Note sure if it matters, though I think we may want this without the trailing slash similar to what admin has.  Guaranteed that it'll work without the trailing slash

https://github.com/silverstripe/silverstripe-admin/blob/1/.gitattributes#L1

Reply:
```suggestion
/tests export-ignore
```